opencda/scenario_testing/utils/sim_api.py

```diff
@@ -187,6 +196,20 @@ def __init__(self, scenario_params,
         # set weather
         weather = self.set_weather(simulation_config['weather'])
         self.world.set_weather(weather)
+        # Define probabilities for each type of blueprint
+        self.use_multi_class_bp = scenario_params[
+            'use_multi_class_bp'] if 'use_multi_class_bp' in scenario_params else False
+        if self.use_multi_class_bp:
```

Comment:
suggest put this chunk of code into a specific function

Reply:
have added comments for better definition of this chunk of code.